### ISSUE TITLE
Consequences of treating 'mitosporic' like 'incertae sedis'

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/reference-taxonomy#153]
I just need to write this down. The problem shows up in OTT 2.9 where a few hundred taxa mentioned in phylesystem, and many in the synthetic tree, have become incertae sedis now that 'mitosporic x' is treated as an incertae sedis container. That makes them unavailable to synthesis.

Major examples are Candida and Phialocephala.

**Comment by jar398** [comment]
premature submission.  issue title and description updated.

**Comment by snacktavish** [comment]
I'm not sure if this is part of the same problem - but I was surprised to find incertae sedis as as taxon in the tree. What is going on there?
https://tree.opentreeoflife.org/opentree/opentree3.0@3448063/Incertae-Sedis

**Comment by jar398** [comment]
Not the same problem. The 'taxa' in question are from SILVA. This bug has
been reported previously and is fixed in OTT 2.9.

**Comment by snacktavish** [comment]
Ah I see. Is there an issue for it somewhere?

**Comment by jar398** [comment]
Yes, it's https://github.com/OpenTreeOfLife/feedback/issues/71   (although it was known & fixed long before that issue was filed)

**Comment by snacktavish** [comment]
Thanks!

**Comment by jar398** [comment]
To get an idea of the magnitude of the problem, I count 1187 taxa that are used in studies, and 141 that are used in the v3 synthetic tree, that are incertae sedis in OTT 2.9, but not in OTT 2.8. (Sorry, that would be 'a thousand', not 'thousands'.) A few of these are due to changes in other taxonomies or their processing, but most are from NCBI. The actual number of affected taxa is of course much bigger, because most taxa do not occur in studies.

Joseph reports that the set of 141 includes all the taxa of at least one source tree.

To see the list, look at the deprecated.tsv file in OTT 2.9, and select lines that say 'newly-hidden' and 'incertae_sedis'. You can also filter on 'synthesis' to get those that are used in v3 synthesis.

**Comment by josephwb** [comment]
Just a note: of 95 fungal studies from Romina, only 75 are not completely nuked by _incertae sedis_ (that is, after pruning such taxa, no taxa remain). These are mostly new studies, so I don't know how many would have gotten through the previous taxonomy (I'll check today). But this certainly argues for letting sampled _incertae sedis_ taxa through.

I feel like this should be communicated some way in the curator, though. Right? Say a curator maps a tree and all/most taxa are marked _incertae sedis_; this might encourage the curator to propose a fix/edit to the taxonomy if they are really good taxa.

**Comment by jar398** [comment]
Re fungal studies, I mistakenly included an old version of Index Fungorum
in OTT 2.9draft9, so perhaps many incertae sedis taxa will be recovered in
2.9draft10. Sorry about that. We should move to draft10 ASAP. This of
course doesn't negate what you say, which I agree with.

There's already a closely related curator app issue:
https://github.com/OpenTreeOfLife/opentree/issues/683

**Comment by kcranston** [comment]
That curator issue is related, but I like the solution that @josephwb proposes. Rather than disallow mapping to incertae sedis descendents (or give an error), it seems that we should allow mapping and then keep those taxa in the tree. It is through phylogeny that we are going to get placement of these taxa, right? Is there some technical hurdle to this solution?

**Comment by josephwb** [comment]
The remaining issue is that if the _entire_ tree consists of _incertae sedis_ taxa (like [this one](https://tree.opentreeoflife.org/curator/study/view/ot_170)), then we do not get placement (unless another tree also samples it).

**Comment by jar398** [comment]
We already allow mapping to incertae sedis (and extinct, etc.) taxa. So
there is no change in that respect. It's just that at present the curator
has no way to know their status, and will think the fact that it's mapped
means it will be in the synthetic tree, when this is not the case.

There is no technical issue to changing the synthesis filtering to include
these taxa.  Just a small amount of programming.

The problem on the UI side (the curator issue I mentioned), like most UI
problems, is more significant, since it involves design.

There is the separate issue, the one Joseph brings up, of linking
subproblems. I suspect that can only be addressed through either better
sampling or manual placement in the taxonomy.

**Comment by josephwb** [comment]
Right - incertae sedis taxa _are_ mapped at the moment.

**Comment by jar398** [comment]
@bredelings Linking you in so you know a bit of the history of this issue.  Nobody has even suggested trying to fix this in the taxonomy (e.g. by making the taxonomy less resolved) so I think I want to pass the baton to you and rely on its being fixed by synthesis.

I had a half-baked idea on how to do this, by the way - it might be possible to represent the lack of information by creating nonstandard splits.  E.g. if ((z,w),((a,b),(c,d),incertae-sedis-e)) then we do have a split z w | incertae-sedis-e, even if we don't know that incertae-sedis-e is not disjoint from (a,b) or (c,d).
